docs/docsite/rst/administration/instances.rst

```diff
@@ -149,15 +151,15 @@ An instance has several attributes that may be configured:
 
```

Comment:
Before the Add Instance procedure, need to add scenario for ingress mesh for "peering into" and include another diagram to depict the ingress scenario (remove the standalone section for mesh ingress).